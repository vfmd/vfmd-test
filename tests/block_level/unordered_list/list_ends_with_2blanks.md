 * First list

 * First list


 * Second list

 * Second list

    Part of the list

 * Second list

 * Second list


    Code block
