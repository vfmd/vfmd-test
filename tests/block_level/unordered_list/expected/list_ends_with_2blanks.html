<ul>
<li><p>First list</p></li>
<li><p>First list</p></li>
</ul>
<ul>
<li><p>Second list</p></li>
<li><p>Second list</p>

<p>Part of the list</p></li>
<li><p>Second list</p></li>
<li><p>Second list</p></li>
</ul>

<pre><code>Code block
</code></pre>
